docs/examples/modules/AccordionExample/index.js

```diff
@@ -1,6 +1,5 @@
 import Accordion from './Accordion.example';
 import AccordionStyled from './AccordionStyled.example';
-import AccordionFluid from './AccordionFluid.example';
```

Comment:
I think we can remove the file too at this point

Reply:
Done